docs/variants/protectli_vp46xx/hardware-matrix.md

```diff
@@ -54,7 +54,31 @@ coreboot port on the Protectli VP46XX firewall.
 | **Power Control**              | RTE v1.1.0                                  |
 | **Power supply**               | Channel Well Technology 12V, 7.5A 90W       |
 
-## Protectli VP4670
+## Protectli VP4670 ver1
+
+| Component                      | Description                                 |
+|--------------------------------|---------------------------------------------|
+| **CPU**                        | Intel Core i7-10810U                        |
+| **RAM**                        | Slot 1: Empty                               |
+|                                | Slot 2: Kingston KVR26S19D8/32              |
+| **Flash memory**               | Macronix KH25L12835F                        |
+| **SSD**                        | SAMSUNG 870 QVO SATA 2.5" SSD 1TB           |
+| **MMC drive**                  | SAMSUNG KLM8G1GETF-B041 (on-board)          |
+| **USB Keyboard**               | DELL KB216                                  |
+| **USB Mouse**                  | DELL MS116p                                 |
+| **Wireless card**              | JJPLUS JWW6051                              |
+| **4G Modem**                   | AMIT MDG200-0T001                           |
+| **Display**                    | Display 1: HDMI 1920x1080p                  |
+|                                | Display 2: DP 1920x1080p                    |
+| **Ethernet controller**        | 6x intel i225 (on-board)                    |
+| **Internal devices**           | Audio subsystem                             |
+| **TPM**                        | Protectli TPM                               |
+| **Network**                    | Local network wired connection              |
+| **Remote Testing Environment** | RTE v1.1.0                                  |
+| **Power Control**              | RTE v1.1.0                                  |
```

Comment:
As power control should be set Sonoff S26. The same in the `Protectli VP4670 ver2` config

Reply:
Changed to:
| **Power Control**              | Sonoff S26                                  |